crates/forge/tests/it/zk/repros.rs

```diff
@@ -0,0 +1,67 @@
+//! Forge tests for zkysnc issues, to avoid regressions.
+//!
+//! Issue list: https://github.com/matter-labs/foundry-zksync/issues
+
+use crate::{
+    config::*,
+    repros::test_repro,
+    test_helpers::{ForgeTestData, TEST_DATA_DEFAULT},
+};
+use alloy_primitives::Address;
+use foundry_config::{fs_permissions::PathPermission, FsPermissions};
+use foundry_test_utils::Filter;
+
+//zk-specific repros configuration
+async fn repro_config(
+    issue: usize,
+    should_fail: bool,
+    sender: Option<Address>,
+    test_data: &ForgeTestData,
+) -> TestConfig {
+    foundry_test_utils::init_tracing();
+    let filter = Filter::path(&format!(".*repros/Issue{issue}.t.sol"));
+
+    let mut config = test_data.config.clone();
+    config.fs_permissions = FsPermissions::new(vec![
+        PathPermission::read("./fixtures/zk"),
+        PathPermission::read("zkout"),
+    ]);
+    if let Some(sender) = sender {
+        config.sender = sender;
+    }
+
+    let runner = test_data.runner_with_zksync_config(config);
+    TestConfig::with_filter(runner, filter).set_should_fail(should_fail)
+}
+
+// https://github.com/matter-labs/foundry-zksync/issues/497
+test_repro!(497);
+
+// https://github.com/matter-labs/foundry-zksync/issues/478
```

Comment:
I do wonder if it adds value to address an issue like this in tests vs just having a test that says `zkevm_call_to_non_existent_address_triggers_a_warning` which would remove a layer of indirection

Reply:
Maybe we should check foundry tests to see if they only use `test_issue_xxx` format for hard to describe issues. If not we can go with @elfedy's suggestion

Reply:
They tend to do that for repros specifically, and give better descriptions for the rest - we can do both, or at least document the test without having to open the issue always

Reply:
Based on your other comment, I believe this doesn't feel into the category of a repro issue fix. Basically, we didn't fix anything since this was not a bug in the system - perhaps that's why it doesn't fit into the category of "repro passes" macro. Instead we added a new feature to print errors when a certain condition occurs. 

Think this test can be a normal `test_zk_` instead